server_manager/messages/master_messages.json

```
@@ -1163,6 +1163,224 @@
+    "description": "80%+ used",
+    "message": ">80% used"
+  },
+  "server_view_access_keys_add_key_action": {
+    "description": "Add new key",
+    "message": "Create key"
+  },
+  "server_view_server_metrics_tab": {
+    "description": "Tab title",
+    "message": "Server Metrics"
+  },
+  "server_view_server_metrics_privacy_statement": {
+    "description": "Privacy line",
+    "message": "We are committed to protecting the privacy and security of our users. Each Outline server automatically collects aggregated and anonymized usage data. This data does not include information about the websites users visit or their communications. It is also not shared with the Outline team unless you"
+  },
+  "server_view_server_metrics_bandwidth_title": {
+    "description": "Total bandwidth used",
+    "message": "Total bandwidth usage $START_OF_ITALICS$(last 30 days)$END_OF_ITALICS$",
+        "placeholders": {
+      "END_OF_ITALICS": {
+        "content": "{closeItalics}"
+      },
+      "START_OF_ITALICS": {
+        "content": "{openItalics}"
+      }
+    }
+  },
+  "server_view_server_metrics_bandwidth_tooltip": {
+    "description": "Total bandwidth tooltip",
+    "message": "This shows the total amount of data transferred through the server over the last 30 days."
+  },
+  "server_view_server_metrics_bandwidth_usage": {
+    "description": "Current bandwidth usage",
+    "message": "Current bandwidth usage"
+  },
+  "server_view_server_metrics_bandwidth_usage_max": {
+    "description": "Max bandwidth used",
+    "message": "Maximum bandwidth usage $START_OF_ITALICS$(last 30 days)$END_OF_ITALICS$",
+        "placeholders": {
+      "END_OF_ITALICS": {
+        "content": "{closeItalics}"
+      },
+      "START_OF_ITALICS": {
+        "content": "{openItalics}"
+      }
+    }
+  },
+  "server_view_server_metrics_bandwidth_as_breakdown": {
+    "description": "Total bandwidth AS breakdown",
+    "message": "ASes with most bandwidth usage $START_OF_ITALICS$(last 30 days)$END_OF_ITALICS$",
+        "placeholders": {
+      "END_OF_ITALICS": {
+        "content": "{closeItalics}"
+      },
+      "START_OF_ITALICS": {
+        "content": "{openItalics}"
+      }
+    }
+  },
+  "server_view_server_metrics_tunnel_time_title": {
+    "description": "Total Tunnel Time",
+    "message": "Total Tunnel Time $START_OF_ITALICS$(last 30 days)$END_OF_ITALICS$",
+        "placeholders": {
+      "END_OF_ITALICS": {
+        "content": "{closeItalics}"
+      },
+      "START_OF_ITALICS": {
+        "content": "{openItalics}"
+      }
+    }
+  },
+  "server_view_server_metrics_tunnel_time_tooltip": {
+    "description": "Tunnel Time tooltip",
+    "message": "This shows the total time users were connected to the server over the last 30 days. It is calculated by adding up the connection time (Tunnel Time) for each device."
+  },
+  "server_view_server_metrics_tunnel_time_as_breakdown": {
+    "description": "Tunnel Time ASl breakdown",
```

```suggestion
    "description": "Tunnel Time AS breakdown",
```

But also add AS info.